packages/theme-chalk/src/common/var.scss

```diff
@@ -10,14 +10,27 @@
 
 /* Transition
 -------------------------- */
-$--all-transition: all 0.3s cubic-bezier(0.645, 0.045, 0.355, 1) !default;
-$--fade-transition: opacity 300ms cubic-bezier(0.23, 1, 0.32, 1) !default;
-$--fade-linear-transition: opacity 200ms linear !default;
-$--md-fade-transition: transform 300ms cubic-bezier(0.23, 1, 0.32, 1),
-  opacity 300ms cubic-bezier(0.23, 1, 0.32, 1) !default;
-$--border-transition-base: border-color 0.2s
-  cubic-bezier(0.645, 0.045, 0.355, 1) !default;
-$--color-transition-base: color 0.2s cubic-bezier(0.645, 0.045, 0.355, 1) !default;
```

Comment:
Are these `0.2s` -> `0.3s`?

Besides, `300ms` or `0.3s` which is better? 

> I see that bootstrap uses `0.2s`.

Reply:
I'm not sure since I have referenced from another project and I think integer here should be more precise than float, even though JavaScript doesn't have a very clear boundary on this.
Also the `0.2` was intentionally changed to `0.3` but I thought this could be a bad thing, do you have any suggestion of naming `0.2s` duration?

Reply:
`0.2s` or `200ms`, I have no idea, I checked several UI libraries, such as `bootstrap`, `vuetify`, `ant-design`.
They all use the form of `0.2s` instead of `200ms`. I think we can keep the same with other UI libraries.

In the jQuery era, I remember `slow` `normal` `fast` to describe animation.

- fast: 200ms
- normal: 400ms
- slow: 600ms

So I think we can reserve three variables `--el-transition-duration-fast` `--el-transition-duration-normal` `--el-transition-duration-slow`.

Of course we can do this first, and I will deal with this part later.